File: week1/mario/more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void) {
    int x = -1;
    while(x < 0 || x > 23) {
        printf("Height: ");
        x = get_int();
    }

    for (int i = 0; i < x; i++) {
        for (int j = x - 1 - i; j > 0; j--) {
            printf(" ");
        }
        for (int j = 0; j < 1 + i; j++) {
            printf("#");
        }
        printf("  ");
        for (int j = 0; j < 1 + i; j++) {
            printf("#");
        }
        printf("\n");
    }
}

File: week8/mario/less/mario.py
import cs50

print("Height: ", end="")
n = cs50.get_int()

for i in range(1 , n + 1):
    for j in range(n - i):
        print(" ", end="")
    for j in range(1 + i):
        print("#", end="")
    print()